PilotDataPlatform/metadata-event-handler
# PILOT-1554: Modify consumer to handle messages from metadata service producer

## Summary

- Modified consumer to handle messages from metadata service producer (`metadata.items` topic).
- The consumer handles all CRUD events from metadata service (including delete actions).

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-1554

## Type of Change


## Testing

Are there any new or updated tests to validate the changes?


## Test Directions

1. Spin up this consumer.
2. Trigger a CRUD event by the metadata service.
3. Check the elastic search for translation of this event into `metadata-items` index.
